esx_service/vmdk_ops.py

```diff
@@ -623,11 +623,11 @@ def findDeviceByPath(vmdk_path, vm):
 
 # Find the PCI slot number
 def get_controller_pci_slot(vm, pvscsi, key_offset):
-    if pvscsi[0].slotInfo:
-       return str(pvscsi[0].slotInfo.pciSlotNumber)
+    if pvscsi.slotInfo:
```

Comment:
since the param types changed., please add a small docstring with param description